_config.yml

```diff
@@ -23,6 +23,11 @@ defaults:
       githubbranch: "master"
       docsbranch: "master"
       versions:
+        - fullversion: "v1.8.0"
+          version: "v1.8"
+          githubbranch: "release-1.8"
+          docsbranch: "release-1.8"
+          url: https://kubernetes.io/docs/home/
         - fullversion: "v1.7.3"
```

Comment:
For now, set fullversion to `v1.7.6` as per the most current version in the [CHANGELOG](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md).

Also, set the *url* for v1.7.x to `https://kubernetes-io-v1-7.netlify.com/docs/home/` until we can get @thockin to setup a subdomain.

Reply:
Done